src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -279,7 +300,7 @@ class $class extends $parentClass
     {
         public function getCacheDir()
         {
-            return $cacheDir;
+            return \$this->getWarmupDir() ?: $cacheDir;
```

Comment:
It isn't. This is an heredoc so unescaped `$` means php will try to interpret `$this` as a variable.